src/components/main.jsx

```
@@ -94,32 +94,31 @@
-        <div className="filter-bar">
-            <h2 className="main--filter">Check them out! : </h2>
+      <main className="flex justify-center items-center w-full">
+        <div className="flex justify-center items-center w-full">
+            <h2 className="text-lg font-medium mr-4">Check them out! : </h2>
             {randomBreeds.map((breed, index) => (
               <button
                 key={index}
                 onClick={() => handleBreedClick(index, breed)}
-                className={index === activeIndex ? 'breed-button-active' : 'breed-button'}
+                className={index === activeIndex ? 'bg-yellow-500 text-white border border-yellow-500 rounded-2xl p-2.5 cursor-pointer transition-colors duration-300' : 'bg-white text-yellow-500 border border-yellow-500 rounded-2xl p-2.5 cursor-pointer transition-colors duration-300 hover:bg-yellow-300 hover:text-white'}
               >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)